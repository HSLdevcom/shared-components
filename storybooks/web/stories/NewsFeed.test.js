import initStoryshots from '@storybook/addon-storyshots';

// eslint-disable-next-line no-undef
jest.mock('../src/Touchable/Touchable');

initStoryshots({
  configPath: 'storybook_web',
  storyKindRegex: /^NewsFeed$/
});
