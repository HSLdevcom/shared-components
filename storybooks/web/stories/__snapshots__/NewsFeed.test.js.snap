// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots NewsFeed default 1`] = `
<div
  className="sc-ksYbfQ jHwjmi sc-cJSrbW ekaOOx"
>
  <ul
    className="sc-kgAjT cuzzPp"
  >
    <li
      className="sc-hqyNC keRCOE sc-kvZOFW cBOBLl"
    >
      <div
        className="text-container sc-frDJqD gCEYuZ"
      >
        <span
          className="top sc-EHOje jvzENE"
        >
          <span
            className="category sc-EHOje jvzENE"
          >
            Talviaikataulut
          </span>
          <span
            className="sc-EHOje jvzENE"
          >
             — 
          </span>
          <span
            className="sc-EHOje jvzENE"
          >
            Today at 2:00 PM
          </span>
        </span>
        <span
          className="bottom sc-EHOje jvzENE"
        >
          HSL:n talviaikataulut tulevat voimaan 15.8 - muutoksia useille linjoille
        </span>
      </div>
      <img
        alt="news image"
        className="sc-hmzhuo eRtqMp"
        src="https://placehold.it/125x521"
      />
    </li>
    <li
      className="sc-hqyNC keRCOE sc-kvZOFW cBOBLl"
    >
      <div
        className="text-container sc-frDJqD gCEYuZ"
      >
        <span
          className="top sc-EHOje jvzENE"
        >
          <span
            className="category sc-EHOje jvzENE"
          >
            Uusi linja
          </span>
          <span
            className="sc-EHOje jvzENE"
          >
             — 
          </span>
          <span
            className="sc-EHOje jvzENE"
          >
            Today at 2:00 PM
          </span>
        </span>
        <span
          className="bottom sc-EHOje jvzENE"
        >
          Uusi raitiolinja 5, Rauttatientori-Katajanokka, aloittaa 20.6
        </span>
      </div>
      <img
        alt="news image"
        className="sc-hmzhuo eRtqMp"
        src="https://placehold.it/521x125"
      />
    </li>
    <li
      className="sc-hqyNC keRCOE sc-kvZOFW cBOBLl"
    >
      <div
        className="text-container sc-frDJqD gCEYuZ"
      >
        <span
          className="top sc-EHOje jvzENE"
        >
          <span
            className="category sc-EHOje jvzENE"
          >
            Talviaikataulut
          </span>
          <span
            className="sc-EHOje jvzENE"
          >
             — 
          </span>
          <span
            className="sc-EHOje jvzENE"
          >
            Today at 2:00 PM
          </span>
        </span>
        <span
          className="bottom sc-EHOje jvzENE"
        >
          HSL:n talviaikataulut tulevat voimaan 15.8 - muutoksia useille linjoille
        </span>
      </div>
      <img
        alt="news image"
        className="sc-hmzhuo eRtqMp"
        src="https://placehold.it/500x500"
      />
    </li>
    <li
      className="sc-hqyNC keRCOE sc-kvZOFW cBOBLl"
    >
      <div
        className="text-container sc-frDJqD gCEYuZ"
      >
        <span
          className="top sc-EHOje jvzENE"
        >
          <span
            className="category sc-EHOje jvzENE"
          >
            Uusi linja
          </span>
          <span
            className="sc-EHOje jvzENE"
          >
             — 
          </span>
          <span
            className="sc-EHOje jvzENE"
          >
            Yesterday at 2:00 PM
          </span>
        </span>
        <span
          className="bottom sc-EHOje jvzENE"
        >
          Uusi raitiolinja 5, Rauttatientori-Katajanokka, aloittaa 20.6
        </span>
      </div>
      <img
        alt="news image"
        className="sc-hmzhuo eRtqMp"
        src="https://placehold.it/50x50"
      />
    </li>
    <li
      className="sc-hqyNC keRCOE sc-kvZOFW cBOBLl"
    >
      <div
        className="text-container sc-frDJqD gCEYuZ"
      >
        <span
          className="top sc-EHOje jvzENE"
        >
          <span
            className="category sc-EHOje jvzENE"
          >
            Talviaikataulut
          </span>
          <span
            className="sc-EHOje jvzENE"
          >
             — 
          </span>
          <span
            className="sc-EHOje jvzENE"
          >
            Yesterday at 2:00 PM
          </span>
        </span>
        <span
          className="bottom sc-EHOje jvzENE"
        >
          HSL:n talviaikataulut tulevat voimaan 15.8 - muutoksia useille linjoille
        </span>
      </div>
    </li>
  </ul>
</div>
`;
